Sources/TuistKit/Utils/SimpleGraphLoader.swift

```diff
@@ -0,0 +1,106 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistLoader
+import TuistPlugin
+import TuistSupport
+
+/// A utility for loading a graph for a given Manifest path on disk
+///
+/// - Any configured plugins are loaded
+/// - All referenced manifests are loaded
+/// - All manifests are concurrently transformed to models
+/// - A graph is loaded from the models
+///
+/// - Note: This is a simplified implementation that loads a graph without applying any mappers or running any linters
+protocol SimpleGraphLoading {
```

Comment:
Naming is hard 😅

Reply:
Lol, I've been thinking about it for a while but I can't come
up with a good name. What about `GraphInstantiator` or `GraphWithPluginsLoader`?

Reply:
maybe bring back a previously used name `ManifestGraphLoader` or `ManifestWithPluginsGraphLoader`?